src/plugins/geoservices/osm/qgeotilefetcherosm.cpp

```diff
@@ -59,17 +59,17 @@ QGeoTiledMapReply *QGeoTileFetcherOsm::getTileImage(const QGeoTileSpec &spec)
     switch (spec.mapId()) {
         case 1:
             // opensteetmap.org street map
-            request.setUrl(QUrl(QStringLiteral("http://otile1.mqcdn.com/tiles/1.0.0/map/") +
+            request.setUrl(QUrl(QStringLiteral("http://korona.geog.uni-heidelberg.de/tiles/roads/") +
```

Comment:
This doesn't seem to work either, I get NXDOMAIN for korona.geog.uni-heidelberg.de

Reply:
I'v set the url to the canonical which makes more sense anyway. Sorry for the confusion.